Andriy_Povzun/Task1.py

```diff
@@ -0,0 +1,6 @@
+employees_eleks = ['John Jones', 'Andy Ruis', 'Oleksandr Usyk',
+                   'Andrii Yevtushenko', 'Will Smith', 'Conor McGregor']
+employees_toshiba = ['Dana White', 'Kamaru Usman', 'George St-Pierre',
+                     'John Jones', 'Antony Joshua', 'Muhammad Ali']
+employees_toshiba.extend(employees_eleks)
+print(employees_toshiba)
```

Comment:
No new line at the end.
Eleks still have employees , please fix it

Reply:
Done.